scripts/live_test/sendemail.py

```diff
@@ -26,6 +26,10 @@
 ACCOUNT_KEY = sys.argv[9]
 COMMIT_ID = sys.argv[10]
 DB_PWD = sys.argv[11]
+DB_USER = sys.argv[12]
+DB_HOST = sys.argv[13]
+DB_PORT = sys.argv[14]
+DB_SCHEME = sys.argv[15]
```

Comment:
This is a really fragile implementation to index the `argv` list, but let's keep it as is at the moment.